### PR TITLE
Added more resources that I get recommended often

### PR DESCRIPTION
**Pull request by EllisBarnes00**
I have been using some of these resources as I program in Java, and they are very well documented.

### CODE REVIEW

File: lists/opengl.md [hunk 1]
```diff
@@ -30,16 +30,22 @@
...
 ### Tutorials
-
-
-
+- #### [OpenGL 3D Game Tutorials](https://www.youtube.com/playlist?list=PLRIWtICgwaX0u7Rf9zkZhLoLuZVfUksDP)
+	- Java OpenGL tutorials, can easily be translated to another programming language.
+- #### [3D Game Development with LWJGL 3](https://lwjglgamedev.gitbooks.io/3d-game-development-with-lwjgl/content/)
+	- LWJGL3 Game Engine tutorial (has source code on GitHub as well).
+- ### [Learn OpenGL](https://learnopengl.com/)
+	- A popular website to get started learning GLFW and OpenGL in C++, however it can easily be translated into other languages.
```
Comment (Jackojc) [inline lists/opengl.md]: I'd rather you drop both Learn "OpenGL" and "3D Game Development with LWJGL 3". I've been informed by other members who contribute to the OpenGL list that they may not be great resources. Drop those and I'll merge the others.

File: lists/opengl.md [hunk 2]
```diff
@@ -30,16 +30,22 @@
 
 - #### [OpenGL 4 Reference](https://www.khronos.org/registry/OpenGL-Refpages/gl4//) [N/A]
 	- Quick reference index for modern OpenGL including version 3.3. Also contains references for GLSL.
+	
+- #### [OpenGL Programming Guide: The Official Guide to Learning OpenGL, Version 4.5 with SPIR-V (9th Edition) ](https://www.amazon.com/OpenGL-Programming-Guide-Official-Learning/dp/0134495497)
```
Comment (Jackojc) [inline lists/opengl.md]: Random space at the end of: " (9th Edition) ]" <- here.

File: lists/opengl.md [hunk 3]
```diff
@@ -30,16 +30,22 @@
...
 
 
 
 
 
 ### Tutorials
-
-
-
+- #### [OpenGL 3D Game Tutorials](https://www.youtube.com/playlist?list=PLRIWtICgwaX0u7Rf9zkZhLoLuZVfUksDP)
```
Comment (Jackojc) [inline lists/opengl.md]: Don't forget to include difficulty level alongside link. "[OpenGL 3D Game Tutorials](https://www.youtube.com/playlist?list=PLRIWtICgwaX0u7Rf9zkZhLoLuZVfUksDP) [Beginner]", Difficulty can be either Beginner/Intermediate/Advanced or N/A.

**Comment by EllisBarnes00** [comment]
I went ahead and fixed those issues, can I ask why [3D Game Development with LWJGL 3](https://www.gitbook.com/book/lwjglgamedev/3d-game-development-with-lwjgl/details) is a bad resource?

**Comment by Jackojc** [comment]
It's a little bit more geared towards Java than I'd like, We're trying to keep this list as language agnostic as possible.

I'm going to ahead and merge this now, Thanks! ;)

**Pull request merged**